src/components/inputs/Pay/PayInputGroup.tsx

```diff
@@ -121,6 +110,15 @@ export default function PayInputGroup({
           disabled={disabled}
         />
       </div>
+      <PayInputSubText
```

Comment:
For this: 
<img width="542" alt="Screen Shot 2022-08-05 at 2 45 40 pm" src="https://user-images.githubusercontent.com/96150256/183003344-c4578b98-bac7-4333-913a-5132cb42afd2.png">